nevergrad/functions/multiobjective/tests/test_structures.py

```diff
@@ -0,0 +1,98 @@
+from unittest import TestCase
+
+from nevergrad.functions.multiobjective.hypervolume import VectorNode, VectorLinkedList
+
+
+class TestNode(TestCase):
+    def test_initialize(self):
+        dim = 4
+        node = VectorNode(dim)
+
+        self.assertIsNone(node.coordinate)
+        for entry in node.next:
+            self.assertTrue(entry is None)
+        for entry in node.prev:
+            self.assertTrue(entry is None)
+
+        self.assertListEqual(list(node.area), [0.0] * dim)
+        self.assertListEqual(list(node.volume), [0.0] * dim)
+
+
+class TestMultiList(TestCase):
+    def setUp(self) -> None:
```

Comment:
With a setup, it makes more sense to use a TestCase, but then again you could also define it through a pytest ficture: https://docs.pytest.org/en/latest/fixture.html
Up to you for this.